llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.cpp

```diff
@@ -1924,6 +1924,10 @@ bool isChainCC(CallingConv::ID CC) {
   }
 }
 
+bool isCallableCC(CallingConv::ID CC) {
+  return !isEntryFunctionCC(CC) && !isChainCC(CC);
```

Comment:
Why are we excluding the chain CC here?

With a chain CC, the previous function that chain-called us could have used scratch.

Reply:
You're right, thanks!

Reply:
This function is now unused.